git.md

```diff
@@ -37,9 +43,6 @@ Tell git to use it:
 git config --global commit.template ~/.gitmessage
 ```
 
-Whenever you create a new commit, use `git commit -a` (or just `git ca` if you're using our [aliases](https://github.com/cfpb/dotfiles/blob/9342ffb879c3134a9b249fea7c50486679f79a8a/.gitconfig#L35)).
-Git will open the message template in your [default editor](https://help.github.com/articles/associating-text-editors-with-git/).
```

Comment:
Sorry if I wasn't clear enough. I would still mention that when you `git commit` that it will open the template in your editor. I would just lave out the `-a` flag and the part about the `ca` alias. I don't want to encourage people to not think about what files they're committing, most of the time.